XamarinCommunityToolkit/Converters/EqualsConverter.shared.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace XamarinCommunityToolkit.Converters
+{
+    /// <summary>
+    /// Checks whether the incoming value equals the provided parameter.
+    /// </summary>
+    public class EqualsConverter : IValueConverter
+    {
+        /// <summary>
+        /// Checks whether the incoming value equals the provided parameter.
+        /// </summary>
+        /// <param name="value">The value to convert.</param>
+        /// <param name="targetType">The type of the binding target property.</param>
+        /// <param name="parameter">Additional parameter for the converter to handle. Not implemented.</param>
+        /// <param name="culture">The culture to use in the converter.</param>
+        /// <returns>An inverted boolean from the one coming in.</returns>
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture) => value.Equals(parameter);
```

Comment:
I wonder, should we use ```culture``` parameter for strings comparison? Not it's ignored.

Reply:
I guess the question also is; do we start a completely new means of checking for string equality within this converter or do we start a different one for it? And are there other types that need "special treatment" that we might need to take into consideration?

Reply:
I'm not sure either, but... we don't need to make a decision right here and now. If the need arises in the community people can request this kind of stuff and/or add to it themselves right? If the need arises we can act on it then.